lib/remote_census_api.rb

```diff
@@ -100,12 +100,11 @@ def request(document_type, document_number, date_of_birth, postal_code)
 
     def fill_in(structure, path_value, value)
       path = parse_path(path_value)
-
       update_value(structure, path, value) if path.present?
     end
 
     def parse_path(path_value)
-      path_value.split(".").map { |section| section.to_sym } if path_value.present?
+      path_value.split(".") if path_value.present?
```

Comment:
This change is necessary because we now use strings instead of symbols, right?

By the way, I've just noticed the method `parse_path` is defined twice on this file. I wonder why our `Lint/DuplicateMethods` rule didn't catch it :thinking:.

Reply:
Yes @javierm , now we use strings instead of symbols.
I think the `Lint/DuplicateMethods` rule doesn't detect it because although it's in the same file it's not in the same class? 🤔 
Anyway I will rename the methods to make it clearer (`parse_response_path`, `parse_request_path`)

Reply:
> although it's in the same file it's not in the same class?

Of course :smile:. Didn't realize the `Response` class finished before the `private` keyword :sweat_smile:.